### PR TITLE
Simplify fixtures

### PR DESCRIPTION
**Pull request by matklad**
bors r+
🤖

### CODE REVIEW

**Comment by matklad** [comment]
bors +

File: crates/ra_hir_def/src/nameres/tests.rs [hunk 1]
```diff
@@ -423,31 +423,6 @@ fn extern_crate_rename_2015_edition() {
...
-        }
-
-        ",
-    );
-
-    assert_snapshot!(map, @r###"
-        ⋮crate
-        ⋮C: t v
-    "###);
-}
```
Comment (bjorn3) [inline crates/ra_hir_def/src/nameres/tests.rs:449]: Why was this removed?

Comment (matklad) [reply crates/ra_hir_def/src/nameres/tests.rs:449]: That was a single test testing the source root functionality, it seems like it’s worth it to remove the test at the cost of simplifying stuff outside. It’s unlikely that it’ll catch any real bugs
